contracts/TransferLimitModule.sol

```diff
@@ -0,0 +1,417 @@
+pragma solidity 0.4.24;
+
+import "../base/Module.sol";
+import "../base/ModuleManager.sol";
+import "../base/OwnerManager.sol";
+import "../common/Enum.sol";
+import "../common/SignatureDecoder.sol";
+import "../common/SecuredTokenTransfer.sol";
+
+import "@gnosis.pm/dx-contracts/contracts/DutchExchange.sol";
+import "@gnosis.pm/dx-contracts/contracts/Oracle/PriceOracleInterface.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+/// @title Transfer Limit Module - Allows to transfer limited amounts of ERC20 tokens and Ether.
+contract TransferLimitModule is Module, SignatureDecoder, SecuredTokenTransfer {
+    string public constant NAME = "Transfer Limit Module";
+    string public constant VERSION = "0.0.2";
```

Comment:
Let's use a version specific to the module (not to the Safe ;) ). So either 0.0.1 or 0.1.0 oder 1.0.0 ... up to you ;)